data/versions.json

```diff
@@ -864,6 +876,18 @@
     },
     "checksums": "https://get.pulumi.com/releases/sdk/pulumi-3.31.1-checksums.txt"
   },
+  {
+    "version": "v3.31.1-ci-test.0",
```

Comment:
This is strange. Not sure why this is showing up. But I don't think it'll cause any harm.